cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -166,7 +166,9 @@ def set_app_version():
 @when('kube-dns.available', 'kubernetes-master.components.installed')
 def idle_status():
     ''' Signal at the end of the run that we are running. '''
-    if hookenv.config('service-cidr') != service_cidr():
+    if not all_kube_system_pods_running():
+        hookenv.status_set('waiting', 'Waiting for kube-system pods to start')
+    elif hookenv.config('service-cidr') != service_cidr():
```

Comment:
I missed where we are declaring service_cidr(), this must have been a prior update that I missed where its now a method.